packages/api-request-builder/src/build-query-string.js

```diff
@@ -1,7 +1,7 @@
 /* @flow */
 import type {
   ServiceBuilderDefaultParams,
-} from 'types/sdk'
+} from '../../../types/sdk'
```

Comment:
This was working fine, there is an alias resolver for that (otherwise linting on master will fail). Are you sure you have the correct dependencies installed?